# description : "pytest plugin for working with MPI"
# depends : python-pytest

_pkg=pytest-mpi
name=python-${_pkg}
version=0.4
release=3
url="https://pytest-mpi.readthedocs.io/"
make# depends : python-setuptools
check# depends : openmpi python-mpi4py python-sybil
source=https://files.pythonhosted.org/packages/source/${_pkg::1}/${_pkg}/${_pkg}-${version}.tar.gz
#        pytest-mpi-pytest-6.patch::https://patch-diff.githubusercontent.com/raw/aragilar/pytest-mpi/pull/12.patch
#source=${url}/archive/v${version}/${name}-${version}.tar.gz
sha256sums=8879c82a9e837bdb49a1675858a13ef22e61e379c983fd37491f876c299b8b31
#            0803b5c675206dd36056bcd91dfbd234f4147afe76cf6dd124e36ec358c49557

#prepare {
#   cd ${_pkg}-${version}
#  patch -Np1 -i ../pytest-mpi-pytest-6.patch
#}

build()-delete {
   cd ${_pkg}-${version}
  python setup.py build
}

# Errors everywhere, similar to https://github.com/browsertron/pytest-parallel/issues/62
#check {
#   cd ${_pkg}-${version}
#  pytest tests -p pytester --runpytest=subprocess
#}

options="!checksum"

build() {
   cd ${_pkg}-${version}
  python setup.py build
  python setup.py install --root=$PKG --optimize=1
}
